runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -434,4 +435,29 @@ public static void postItemComposition(ItemComposition itemComposition)
 		event.setItemComposition(itemComposition);
 		eventBus.post(event);
 	}
+
+	/**
+	 * Called after a hitsplat has been processed on an actor.
+	 * Note that this event runs even if the hitsplat didn't show up,
+	 * i.e. the actor already had 4 visible hitsplats.
+	 *
+	 * @param actor The actor the hitsplat was applied to
+	 * @param value The value of the hitsplat (i.e. how high the hit was)
+	 * @param type The hitsplat type (i.e. color)
+	 * @param var3
+	 * @param var4
+	 * @param gameCycle The gamecycle the hitsplat was applied on
+	 * @param duration The amount of gamecycles the hitsplat will last for
+	 */
+	public static void onActorHitsplat(Actor actor, int type, int value, int var3, int var4,
```

Comment:
argument order here doesn't match javadoc

Reply:
Fixed this by moving the params in the doc